dataactbroker/permissions.py

```diff
@@ -20,6 +20,10 @@ def decorated_function(*args, **kwargs):
                     for permission in permissionList :
                         if(not userDb.hasPermission(user, permission)) :
                             validUser = False
+                        else:
```

Comment:
If I read this correctly, it looks like this code changes the way this decorator works when we're checking for multiple permissions....from "user needs to have all permissions in the list to be considered valid" to "user is valid if they have at least one permission in the list." Is this the intended effect?